### ISSUE TITLE
Do you still working on this

### ISSUE THREAD

**Issue by paja-talic** [salimvanak/myWMS#2]
Do you still working on tgis project

**Comment by salimvanak** [comment]
Yes. This version is still in use.  But it does need a reasonable upgrade if the java vms and the wildly server.  I just haven't had the time to do it
